src/common.speech/DialogServiceAdapter.ts

```diff
@@ -614,8 +614,8 @@ export class DialogServiceAdapter extends ServiceRecognizerBase {
                 break;
 
             default:
-                // tslint:disable-next-line:no-console
-                console.log("Unexpected response of type " + responsePayload.messageType + ". Ignoring.");
+                Events.instance.onEvent(
+                    new BackgroundEvent("Unexpected response of type '" + responsePayload.messageType + "'. Ignoring."));
```

Comment:
A bit pedantic, but I like the newer backtick interpolated string literals, ala:
```
new BackgroundEvent(`Unexpected response of type '${responsePayload.messageType}'. Ignoring.`));
```

Reply:
That's not pedantic at all and I appreciate you calling it out! I was actually curious if string interpolation was possible and I should have looked more closely. Gladly adopted!